src/utils/hendelseFilteringUtils.tsx

```diff
@@ -312,8 +316,14 @@ const sortEventsOnFrist = (
 ): PersonregisterState => {
   const sorted = Object.entries(personregister).sort(
     ([, persondataA], [, persondataB]) => {
-      const fristDateA = persondataA.aktivitetskravVurderingFrist;
-      const fristDateB = persondataB.aktivitetskravVurderingFrist;
+      const fristDateA =
```

Comment:
Sammenligner tidligste frist per person dersom det er stigende sortering og ellers seneste frist. Litt usikker på om det er en grei måte å gjøre det på, så setter pris på om flere kan teste lokalt for å se om det gir mening.

Reply:
Den synes jeg var vanskelig! Ascending synes jeg gir mening, siden det da er den øversten datoen på hver rad som sammenlignes. Men på descending er det den nederste. Men når det kun er én dato, tenker jeg på den som "øverst" 🤔 Det er jo heller ikke helt intuitivt hvordan sorteringen fungerer med de ulike symbolene 🤷‍♂️

Jeg tror det kan bli rart uansett hvis man vil, men i praksis vil vel kanskje de to datoene være ganske like så lenge begge er satt? Og da vil jo sorteringen være "ca riktig" uansett hvilken man ser på?